README.rst

```diff
@@ -542,14 +542,42 @@ List topologies
 
 .. code-block:: text
 
-    GET /api/v1/topology/
+    GET /api/v1/topology/topology/
```

Comment:
as the prefix for the API URLs, we're using the name of the module without `openwisp-`, so it should be:

```/api/v1/network-topology/topology/```